package.json

```diff
@@ -11,7 +11,7 @@
     "build:docs": "wireit",
     "check:pinned-deps": "tsx tools/ensure-pinned-deps",
     "check": "npm run check --workspaces --if-present && run-p check:*",
-    "clean": "rm -rf **/.wireit && npm run clean --workspaces --if-present",
+    "clean": "rm -rf $(find . -type d -name .wireit) && npm run clean --workspaces --if-present",
```

Comment:
what happens if no .wireit folder found?

Reply:
Decided to use rimraf